src/main/java/org/sonar/squidbridge/annotations/ActivatedByDefault.java

```diff
@@ -24,6 +24,12 @@
 import java.lang.annotation.RetentionPolicy;
 import java.lang.annotation.Target;
 
+/**
+ * @deprecated since 2.7. Will be removed without alternative.
+ * Rules should be assigned to dedicated profiles.And plugins should define quality profiles 
+ * with org.sonar.api.server.profile.BuiltInQualityProfilesDefinition.
+ */
+@Deprecated
 @Retention(RetentionPolicy.RUNTIME)
 @Target(ElementType.TYPE)
 public @interface ActivatedByDefault {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Do not forget to remove this deprecated code someday. ([squid:S1133](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1133))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=org.sonarsource.sslr-squid-bridge%3Asslr-squid-bridge&branch=SSLRSQBR-71&issues=AWHXxloRaf12pdu-6jE5&open=AWHXxloRaf12pdu-6jE5)

[ISSUE_KEY]: AWHXxloRaf12pdu-6jE5